jdaviz/configs/mosviz/tests/test_parsers.py

```diff
@@ -190,8 +190,8 @@ def test_nircam_parser(mosviz_helper, tmp_path):
     # Check that the correct amount of spectra got loaded in the correct order
     assert len(mosviz_helper.app.data_collection) == 31
     assert mosviz_helper.app.data_collection['MOS Table']['Identifier'][0] == 1112
-    assert mosviz_helper.app.data_collection[0].label == 'GRISMR Source 1112 spec2d R'
-    assert mosviz_helper.app.data_collection[15].label == 'GRISMR Source 1112 spec1d R'
+    assert mosviz_helper.app.data_collection[0].label == 'F322W2 Source 1112 spec2d R'
```

Comment:
What does this mean? That the parser has been loading the wrong thing for NIRCam test case all this time? 😱

Reply:
It's loading the same thing, just labeling it differently. Due to differences in what goes in which header keyword between the instruments, NIRCam was trying to match up "GRISMR" from the 2D spectrum to "CLEAR" from the direct image, rather than this filter string like NIRISS does.

Reply:
@camipacifici probably needs to confirm that displaying this filter rather than "GRISMR" in the table is ok. Maybe we want both.

Reply:
Yes, it would be best to have both, but it can be part of a separate PR.